README.md

```diff
@@ -1,4 +1,6 @@
-## MaaS Components
+# MaaS Components
+
+[![Android Build](https://github.com/trafi/maas-components/workflows/Android%20Build/badge.svg?event=push)](https://github.com/trafi/maas-components/actions?query=workflow%3A%22Android+Build%22)
```

Comment:
This badge doesn't currently work, will it? 🙂

Reply:
Should work after merge. I'm 90% sure it's correct :smile:

Reply:
Well that 10% got me :smile: Action appeared before merging once started using `main` branch :pray: